### ISSUE TITLE
Remove any Voyager specific implementation from the GIFM processing logic

### ISSUE THREAD

**Issue by jcreel** [TAMULib/GIFMButtonService#18]
After #73 #74 #76 are completed...

We will need to add a class generalizing CatalogItem, which is currently just a Map.

The GIFM Service will then be able to call predictable methods to get item data.

This may have been done, but we should double check.

**Comment by jsavell** [comment]
The CatalogItem would still be nice to have, but all CatalogService clients are currently functioning fine without it.
